.github/workflows/post-apiview.yml

```
@@ -0,0 +1,22 @@
+name: After APIView
+
+on:
+  check_suite:
+    types: [completed]
+
+permissions: {}
+
+jobs:
+  event-handler:
+    permissions:
+      statuses: write
+      pull-requests: write
+      checks: read
```

```suggestion
permissions:
  statuses: write
  pull-requests: write
  checks: read

jobs:
  event-handler:
```

simpler if all job(s) use same permissions